geo-data-server/src/main/java/com/powsybl/geodata/server/GeoDataService.java

```diff
@@ -50,6 +50,8 @@
         // TODO filter by country
         StopWatch stopWatch = StopWatch.createStarted();
 
+        LOGGER.info("countries size {}", countries.size());
```

Comment:
This should be a trace log not an info.

Reply:
just to avoid sonar major error : unused parameter  countries.
i'll change info -> trace

Reply:
So you can remove the log and keep the Sonar issue. That will be solved once countries will be used in tha Cassandra query.